### PR TITLE
fix-can_not_trace_multi_service_name

### PR DESCRIPTION
**Pull request by SuriyaRuk**
when I want to add a multi-service name APM but the plugin can config only one 
I fix it by initrace always . It works

### CODE REVIEW

**Comment by matthyx** [comment]
Hi @SuriyaRuk thanks for your PR. Can you send me your configuration to better understand the use case?
Maybe we can modify the condition instead of removing it completely?

**Comment by SuriyaRuk** [comment]
I have 3 plugins and routes.
image
I enabled the plugin on 3 plugins and routes in the different APM service names. I enabled 3 times
image
image
image
image

I was facing a bug. The APM server has only one service name: app1

I fixed the code in the main.go: inittrace always this issue was solved.

File: main.go [hunk 1]
```diff
@@ -271,9 +271,9 @@ func (conf Config) Log(kong *pdk.PDK) {
 		return
 	}
 	// (eventually) initialize tracer
```
Comment (matthyx) [inline main.go:273]:
```suggestion
	// initialize tracer
```

File: main.go [hunk 2]
```diff
@@ -271,9 +271,9 @@ func (conf Config) Log(kong *pdk.PDK) {
...
-	if tracer == nil {
+	//if tracer == nil {
```
Comment (matthyx) [inline main.go:274]:
```suggestion
```

File: main.go [hunk 3]
```diff
@@ -271,9 +271,9 @@ func (conf Config) Log(kong *pdk.PDK) {
...
 		initTracer(conf, kong.Log)
```
Comment (matthyx) [inline main.go:275]:
```suggestion
	initTracer(conf, kong.Log)
```

File: main.go [hunk 4]
```diff
@@ -271,9 +271,9 @@ func (conf Config) Log(kong *pdk.PDK) {
...
-	}
+	//}
```
Comment (matthyx) [inline main.go:276]:
```suggestion
```

**Review by matthyx — CHANGES_REQUESTED** [review]
ok then let's completely remove the if and update the comment

**Comment by matthyx** [comment]
thanks for your contribution!

**Pull request merged**